doc/gui/examples/controls/datanode_viewer_json.py

```diff
@@ -0,0 +1,22 @@
+import taipy as tp
+import taipy.gui.builder as tgb
+from taipy import Config, Orchestrator, Scope
+from taipy.gui import Gui
+
+json_config_node = Config.configure_json_data_node(
+    id="json_node",
+    default_path="./demo-1248-dn.json",
```

Comment:
where is the json file ?

Reply:
I guess it should be renamed datanode_viewer_json.json
And that would be why the demo-* file was ignored